Project-Hotpot/Project-Hotpot/ViewControllers/HomeViewController.swift

```diff
@@ -39,6 +43,9 @@ class HomeViewController: ViewController {
         }
     }
     
+    
+    
+    
```

Comment:
Ah invisible code is so beautiful, but don't get too attached :-P. Please remove